Formula/tensorflow_c.rb

```diff
@@ -0,0 +1,46 @@
+class TensorflowC < Formula
+  desc "C interface for Google's OS library for Machine Intelligence"
+  homepage "https://www.tensorflow.org/"
+  url "https://github.com/tensorflow/tensorflow.git", :tag => "v1.0.0",
+    :revision => "07bb8ea2379bd459832b23951fb20ec47f3fdbd4"
+
+  depends_on "bazel" => :build
+  depends_on "pkg-config" => :run
+
+  def install
```

Comment:
Can the release tarballs of the TensorFlow C library be used to avoid building from source?

That is recommended for building language bindings for TensorFlow (for example, the [Go bindings](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/go) and hopefully the Rust bindings will switch to using this binary target when possible too)